buildpipe/pipeline.py

```diff
@@ -155,7 +156,7 @@ def iter_stairs(stairs: List[box.Box], can_autodeploy: bool) -> Generator[box.Bo
 
 def check_autodeploy(deploy: Dict) -> bool:
     now = datetime.datetime.now(pytz.timezone(deploy.get('timezone', 'UTC')))
-    check_hours = re.match(deploy.get('allowed_hours_regex', '\d|1\d|2[0-3]'), str(now.hour))
+    check_hours = re.match(deploy.get('allowed_hours_regex', '\\d|1\\d|2[0-3]'), str(now.hour))
```

Comment:
why's the double backslash needed in the default regexp? I would've thought you'd want something like `(\d|1\d|2[0-3])$`

Reply:
I was getting linting errors using single backslash